### ISSUE TITLE
[Accessibility] Skip to main content link should be the first tab stop on the page 

### ISSUE THREAD

**Issue by elenm** [CityOfDetroit/bloom#467]
Currently the first tab stops are the languages in the upper right hand corner.

**Comment by ludtkemorgan** [comment]
Fixed as part of accessibility audit
